OpenRA.Game/Traits/World/Shroud.cs

```diff
@@ -383,13 +453,25 @@ public bool IsVisible(Actor a)
 
 		public bool IsTargetable(Actor a)
 		{
+			IEnumerable<CPos> vis = GetVisOrigins(a);
+			bool concealed = vis.All(c => generatedDisruptionCount[c] > 0 || generatedShroudCount[c] > 0);
```

Comment:
This code deals with regular actors (not FrozenActors), so I'm not sure what you're asking.